security/security-spring/src/main/resources/applicationContext-security.xml

```diff
@@ -53,14 +53,38 @@
     <http pattern="/reactapp/**" security="none"/>
     <http pattern="/api/swagger-resources/**" security="none"/>
     <http pattern="/api/swagger-ui.html" security="none"/>
-        
+
     <!-- This must come before the default entry point which will capture everything not matched by pattern -->
-    <http use-expressions="true" pattern="/api/**" entry-point-ref="restAuthenticationEntryPoint" create-session="never">
+    <http use-expressions="true" pattern="/api/**" entry-point-ref="restAuthenticationEntryPoint" create-session="never" authentication-manager-ref="tokenAuthenticationManager">
```

Comment:
@mandawilson Doesn't this mean that when this code gets merged, web page requests will need tokens?  Ah wait no, in the filter, you only requiresAuthentication if an Authorization header is present otherwise control moves to the next filter in the framework......nice.

Reply:
Why do you need authentication-manager-ref defined here and within the TokenAuthenticationFilter?

Reply:
It doesn't look like you based these changes on the spring security 5 code - was this intentional?

Reply:
@n1zea144 Yes it isn't based on Spring Security 5 code because those changes hadn't been merged in when I started this.  I will get them now.  I am pretty confident it won't change anything because most the time I was reading security documentation for more recent Spring Security versions and then checking to make sure they hadn't changed since version 3.

I can remove authentication-manager-ref, I left it in after testing it and realizing it wasn't helpful (it didn't stop the second manager from overriding this referenced one).

Reply:
I was about to merge this .. but I will wait for the update to spring-security v5